Dtm update (#112)
SoftwareAG/c8y-releasenotes: content/release-10-16-0/digital-twin-manager-10-16-0-bundle/10-16-0-0.md

<td>CTM-155</td>
</tr>

<tr>

> <td>The DTM application is added with contextual help to guide user on DTM application usage.</td>

Reviewer: ```suggestion
<td>Contextual help has been added to the DTM application to guide users on DTM application usage.</td>
```
